test/forms.test.js

```diff
@@ -2,133 +2,92 @@
 
 import proclaim from 'proclaim';
 import sinon from 'sinon/pkg/sinon';
-import * as fixtures from './helpers/fixtures';
+import formFixture from './helpers/fixtures';
 
 const Forms = require('./../main');
 
 describe('Forms', () => {
-	let dispatch;
-	let inputFields;
 	let formEl;
-	let sandbox;
 	let parentClass;
 	let requiredTextField;
-	let optionalTextField;
-	let dateField;
+	let dateFields;
 
-	beforeEach(() => {
-		sandbox = document.createElement('div');
-		sandbox.innerHTML = fixtures.form;
-		document.body.appendChild(sandbox);
-
-		inputFields = document.body.querySelectorAll('input, textarea, select');
-		dateField = inputFields[0];
-		requiredTextField = inputFields[3];
-		optionalTextField = inputFields[4];
-
-		dispatch = (event, element) => element.dispatchEvent(new Event(event));
-		parentClass = (element, validity) => element.closest('.o-forms-input').classList.contains(`o-forms-input--${validity}`);
+	before(() => {
+		parentClass = (element, modifier) => element.closest('.o-forms-input').classList.contains(`o-forms-input--${modifier}`);
```

Comment:
Element.closest does not exist in IE. we need to add it to the browserFeatures.required array and then should also run the tests in IE by using `obt --browerstack`.